examples/todos-flow/src/__tests__/components/TodoList.test.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React from 'react';
+import * as React from 'react';
```

Comment:
These lines should be left alone. This changes behavior in a subtle way and isn't a best practice.

Reply:
Interesting! How it changes the behaviour? This result was purely from flow-upgrade and can be changed ofc.

Flow docs encourage this way of importing React too:

>Note: We import React as a namespace here with `import * as React from 'react'` instead of as a default with `import React from 'react'`. When importing React as an ES module you may use either style, but importing as a namespace gives you access to React’s utility types.

see: https://flow.org/en/docs/react/components/

Reply:
The primary thing is it changes the import from "just the default please" to "everything including the default." This makes it harder for tree-shaking bundlers to work out what is and is not used from a module. It's subtle, but important in its implications.

Reply:
Right, thanks for the clarification :) Indeed it makes tree shaking problem more difficult to solve.